pkgs/data/icons/candy-icons/default.nix

```
@@ -0,0 +1,31 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "candy-icons";
+  version = "2023-10-03";
```

```suggestion
  version = "unstable-2023-10-03";
```
Upstream doesn't have a release tagged with the date. Make sure to edit the commit message and PR title as well